spt/features/visualizations/imgui/spt_imgui_widgets.cpp

```diff
@@ -61,6 +62,23 @@ int SptImGui::CvarCombo(ConVar& c, const char* label, const char* const* opts, s
 	return val;
 }
 
+bool SptImGui::InputTextInteger(const char* label, const char* hint, long long& val, int radix)
+{
+	Assert(radix == 10 || radix == 16);
+	char buf[24];
+	const char* fmtSpecifier = radix == 10 ? "%" PRId64 : "%" PRIx64;
+	snprintf(buf, sizeof buf, fmtSpecifier, val);
+	buf[sizeof(buf) - 1] = '\0';
```

Comment:
p sure this is redundant and should already be done by snprintf

Reply:
You're right, I can never remember which printf methods add the terminator and which don't so I just toss that in everywhere. Lemme fix.